types/types.go

```diff
@@ -303,10 +303,10 @@ type Cluster struct {
 
 // Auth holds authentication configuration (BASIC, DIGEST, users)
 type Auth struct {
-	Basic       *Basic
-	Digest      *Digest
-	Forward     *Forward
-	HeaderField string
+	Basic       *Basic   `description:"Basic authentication."`
```

Comment:
```
 --web.auth                                 Authentication for dashboard and API                                             (default "true")
 --web.auth.basic                           Basic authentication.                                                            (default "true")
 --web.auth.digest                          Digest authentication.                                                           (default "true")
 --web.auth.forward                         Forward authentication.                                                          (default "true")
```

`default: true` ? how can I set the users ?

Reply:
`default:true`: it's a flaeg bug see #2080